google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -535,7 +538,8 @@ TEST_F(DataTypeIntegrationTest, SelectIntervalFromTimestampDiff) {
           std::chrono::hours(1))};
   std::time_t now_seconds =
       std::chrono::system_clock::to_time_t(std::chrono::system_clock::now());
```

Comment:
I would also suggest not using `now()` (despite my doing so in the suggested code).  It only makes the test non-hermetic.  A fixed time instant would be better.